packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,144 @@
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+            Accordion three content<div slot="heading">Accordion three</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion four content
+            <div slot="heading">Accordion four</div>
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### API
+
+#### FAST Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_ Unchanged
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+#### FAST Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   `error-visible` - boolean
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` buttons. The nimble-button pattern with these styles seems to fit the styling of the accordion, so these styles can be used. The attributes of `block`, `outline`, and `ghost` will most likely be implemented with the name `appearance`. A few styling elements like the background color change on click that is used in the button will have to be removed.
+
+The Figma design also includes a state that is most likely `error`, as it has a red border and red exclamation mark icon. For this state, the styling of the accordion header button could be altered to have a red border. This could be controlled with a boolean attribute such as `error-visible`, which would have a default of "" in the accordion attributes- when necessary, "error-visible" would be added to the accordion attributes and this would make the accordion switch to the red styling.
+
+A css class for the nimble-accordion-item can be used to get a green border (or red for `error-visible`) around the entire accordion (parent and children of accordion).
```

Comment:
Can you expand on exactly what the api of this would look like?
Sound like you are saying `<nimble-accordion-item class="make-it-green">`. If that is the proposal make sure to spell out what the proposed class name is etc.

In this case, that seems like the wrong direction. We have a pattern (you may be familiar with 😉) of having a appearance variant that uses the brand accent color. 

But but in this case, are you sure having a green / accent mode is what the spec is proposing? I looked briefly at the spec and it seems like the different green borders are associated with hover states / keyboard focus states. They do not seem to be different modes of the control. Is that what you are seeing as well?